File: src/main/java/net/igneo/icv/shader/ModShaders.java
package net.igneo.icv.shader;

import net.igneo.icv.shader.postProcessors.BlinkPostProcessor;
import team.lodestar.lodestone.systems.postprocess.MultiInstancePostProcessor;
import team.lodestar.lodestone.systems.postprocess.PostProcessHandler;

public class ModShaders {
    private static void registerShader(MultiInstancePostProcessor shader) {
        PostProcessHandler.addInstance(shader);
    }
    
    public static void register() {
        registerShader(BlinkPostProcessor.INSTANCE);
    }
}

File: src/main/java/net/igneo/icv/shader/ModShaders.kt
package net.igneo.icv.shader

import net.igneo.icv.shader.postProcessors.BlinkPostProcessor
import team.lodestar.lodestone.systems.postprocess.MultiInstancePostProcessor
import team.lodestar.lodestone.systems.postprocess.PostProcessHandler

private fun registerShader(shader: MultiInstancePostProcessor<*>) {
    PostProcessHandler.addInstance(shader)
}

fun registerShaders() {
    registerShader(BlinkPostProcessor.INSTANCE)
}
